src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -334,31 +325,28 @@ public void GetSuppressedIssues_WhenProjectHasModulesAndIssueIsFileLevelAndIsNot
 
             VerifyServiceGetIssues(Times.Exactly(1)); // issues should be fetched on creation
 
-            // We're deliberately faking SonarQube returning paths with \ instead of / which
```

Comment:
This comment is no longer accurate either so the AssertIgnoreScope isn't required for all of the test cases.

However, it is still required for the input parameter in the second case, so I moved the AssertIgnoreScope to just cover that case.